ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Setup
+
+> Please provide the following details, the more info you can provide the
+> better.
+
+*Operating System*: <Linux | Windows | OS X>
+*Node Version*: <4.2 | 6.0 | ...>
+*web-push Version*: <1.0.0>
+
+> Please select any browsers that you are experiencing problems with:
+
+- [ ] Google Chrome
+- [ ] Firefox
```

Comment:
Nit: For consistency, let's change this to `Mozilla Firefox` or `Google Chrome` to `Chrome`.